src/main/java/com/programmers/ticketparis/exception/ExceptionRule.java

```diff
@@ -14,7 +13,9 @@ public enum ExceptionRule {
     BAD_REQUEST(HttpStatus.BAD_REQUEST, "사용자 입력 유효성 검사 실패"),
     NOT_FOUND(HttpStatus.NOT_FOUND, "요청한 URL에 해당하는 리소스를 찾을 수 없음"),
     METHOD_NOT_ALLOWED(HttpStatus.METHOD_NOT_ALLOWED, "허용되지 않은 HTTP Method 요청 발생"),
-    INTERNAL_SERVER_ERROR(HttpStatus.INTERNAL_SERVER_ERROR, "기타 서버 내부 에러 발생");
+    INTERNAL_SERVER_ERROR(HttpStatus.INTERNAL_SERVER_ERROR, "기타 서버 내부 에러 발생"),
+    NOT_EXIST_RESERVATION(HttpStatus.NOT_FOUND, "요청한 예매 ID가 존재하지 않음"),
+    NOT_EXIST_RESERVATION_STATUS(HttpStatus.NOT_FOUND, "존재하지 않는 예약 상태");
```

Comment:
`p4`
enum 코드값은 추가에 용이하도록 아래처럼 사용하면 좋아요~ 

```suggestion
    NOT_EXIST_RESERVATION_STATUS(HttpStatus.NOT_FOUND, "존재하지 않는 예약 상태"),
    ;
```

Reply:
안그래도 매번 추가할 때마다 지우고 다시 만들어주는게 너무 귀찮았는데 이렇게도 되는거였네요!  사소한거지만 확실히 효율이 더 좋아진 것 같아요. 감사합니다! 👍👍 

적용 커밋: [c333d04](https://github.com/prgrms-be-devcourse/BE-04-TICKETPARIS/pull/29/commits/c333d042f94585cc089e31ed919c71a2f4d7c4ea)